feat(apple): Time to Initial display and Time to Full display measurement (#6519)
getsentry/sentry-docs: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

[UIWindow]: https://developer.apple.com/documentation/uikit/uiwindowdidbecomevisiblenotification
[didFinishLaunching]: https://developer.apple.com/documentation/uikit/uiapplication/1622971-didfinishlaunchingnotification

### Time to Initial Display

_(New in version 8.4.0)_

Time to initial display provides insight into how long it takes to your view controller to launch and draw their first frame, by adding a span for a view controller when it is loaded. The SDK sets the span operation to `ui.load.initial-display` and the span description to the view controllers's name followed by `initial display`, for example, `MainViewController initial display`.

The span starts when the view of a view controller is loaded, and there is no other view controller transaction happening at the moment.

The span finishes after the view appeared on the screen.

### Time to Full Display

_(New in version 8.4.0)_

> Time to full display provides insight into how long it would take your view controller to launch and load all of their content, by adding a span for each launch of an view controller. The SDK sets the span operation to `ui.load.full-display` and the span description to the view controllers's name followed by `full display`, for example, `MainActivity full display`.

Reviewer: ```suggestion
Time to full display provides insight into how long it takes your view controller to launch and load all of its content, by adding a span for each launch of a view controller. The SDK sets the span operation to `ui.load.full-display` and the span description to the view controllers' name followed by `full display`, for example, `MainActivity full display`.
```

The span starts when the view of a view controller is loaded, and there is no other view controller transaction happening at the moment.

_Time to full display is disabled by default, but you can enable it by setting:_

```swift {tabTitle:Swift}
import Sentry

SentrySDK.start { options in
    options.dsn = "___PUBLIC_DSN___"
    options.enableTimeToFullDisplay = true
}
```

```objc {tabTitle:Objective-C}
@import Sentry;

[SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
    options.dsn = @"___PUBLIC_DSN___";
    options.enableTimeToFullDisplay = YES;
}];
```

> _The span has to be finished manually, by using:_

Reviewer: `m`: I think we should explain that we can't detect when your UIViewController is fully loaded. Only you, the user, can achieve this. Therefore, you have to manually call the API to get proper statistics.
